pkg/bugreport/run.go

```diff
@@ -123,7 +123,7 @@ func runCmdAndWriteToFile(cmdList []string, outFile string) error {
 	if err != nil {
 		return err
 	}
-	defer outfile.Close() //nolint: errcheck, #nosec G307
```

Comment:
#nosec is a Gosec directive. Why is that being removed because it seems unlreated to golint? Please add G307 back so we clearly know the security warning is disabled?

Reply:
I originally replaced `#nosec G307` with `gosec` to address the following warning: `[runner/nolint] Found unknown linters in //nolint directives: #nosec g307, go-lint`. However, I've discovered the warning can be resolved by removing the space after `errcheck,`.

Reply:
We should be using `// #nosec` to annotate Gosec warnings.

Reply:
@jaellio Could you add the `// #nosec G307` directive back so we can merge this PR to unblock the CI failure? I want to make sure this is not a part of the `//nolint` directive and switch to the `#nosec` directive to quickly trace security warnings that are masked.